1-exercises/B-highlight-words/script.js

```diff
@@ -1,10 +1,41 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+
+  const parentElement = document.querySelector("#content");
+  const selectElement = document.createElement("select");
```

Comment:
Good to see you're using some descriptive names here! This code works, it  looks nice and youy've even chucked some CSS in, formatting is brillant. I would try to make the function shorter for readabilty - maybe make the event handling function a seperate funtion. Obviously also some non-descriptive names